@here/olp-sdk-dataservice-read/lib/client/CatalogsRequest.ts

```diff
@@ -20,38 +20,50 @@
 import { validateBillingTag } from "..";
 
 /**
- * A class that prepare information for calls to getCatalogs from ConfigAPI.
+ * Prepares information for calls to get catalogs from the OLP Config Service.
  */
 export class CatalogsRequest {
     private schemaHrn?: string;
     private billingTag?: string;
 
+    /**
+     *  Gets a schema HERE Resource Name (HRN) for the request.
+     * 
+     * @return The schema HRN.
+     */
     public getSchema(): string | undefined {
         return this.schemaHrn;
     }
 
     /**
-     * Set value of layer schema HRN to use in method getCatalogs from ConfigClient.
-     * If schema is setted, then getCatalogs will return filtered response by the specified layer schema HRN.
-     * If schema is not setted, then filter will return search for all.
-     * @param schemaHrn layer schema HRN
+     * Sets a value of a layer schema HERE Resource Name (HRN) that is used in the `getCatalogs` method of the [[ConfigClient]] instance.
+     * If the schema is set, the `getCatalogs` method returns a response filtered by the specified layer schema HRN.
```

Comment:
returns only catalogs witch have layer/layers with the specific schema

Reply:
Fixed